lib/puppet/functions/accounts_ssh_authorized_keys_line_parser.rb

```diff
@@ -16,7 +16,7 @@
   end
 
   def accounts_ssh_authorized_keys_line_parser_string(str)
-    matched = str.match(%r{((sk-ecdsa-|ssh-|ecdsa-)[^\s]+)\s+([^\s]+)\s+(.*)$})
+    matched = str.match(%r{((sk-ssh-ed25519-|sk-ecdsa-|ssh-|ecdsa-)[^\s]+)\s+([^\s]+)\s+(.*)$})
```

Comment:
Should that trailing dash be there in the `sk-ssh-ed25519-` pattern? I don't have the hardware to run `ssh-keygen -t ed25519-sk`, but looking at `ssh -Q key`, I'd expect the public key to start with `sk-ssh-ed25519`.

Reply:
You are right. My brain probably added it because I saw a - on the other types as well. I've fixed it.